### PR TITLE
Add Faroese language support

### PR DESCRIPTION
**Pull request by nirmalathapa**
Hello Evan,

Please review the addition of Faroese language support in the language list.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+* new: Færøsk support
```
Comment (EvanHahn) [inline HISTORY.md]: This should be moved below the heading.

File: HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+3.20.1 / 2019-09-25
```
Comment (EvanHahn) [inline HISTORY.md]: Could you change this heading to "Unreleased"?

File: package.json [hunk 3]
```diff
@@ -37,7 +38,7 @@
   },
   "devDependencies": {
     "check-ecmascript-version-compatibility": "^0.1.1",
-    "csv-parse": "^4.4.5",
+    "csv-parse": "^4.4.6",
```
Comment (EvanHahn) [inline package.json]: Why was this update needed?

Comment (nirmalathapa) [reply package.json]: it was part of npm audit fix for https://npmjs.com/advisories/1171, i will make this fix as separate PR

Comment (EvanHahn) [reply package.json]: I'll update all of the devDependencies separately—no need to make a separate PR.

**Comment by EvanHahn** [comment]
Thanks! I'll publish this soon.

(I was having some trouble publishing, so I'll give it a try sometime in the next few days.)

**Comment by EvanHahn** [comment]
This has been deployed in `humanize-duration@3.21.0`! Thanks again.

**Pull request merged**